hydrus/tests/test_app.py

```diff
@@ -412,6 +415,87 @@ def test_endpointClass_GET(self):
                         assert "@id" in response_get_data
                         assert "@type" in response_get_data
 
+    def test_socketio_POST_updates(self):

```

Comment:
The test should also use a socket to connect to the sync mechanism, rather than using an HTTP client.